components/archiver/gtar/patches/revert_check_return_value_from_xgetcwd.patch

```diff
@@ -0,0 +1,59 @@
+The patch was taken from omnios-build:
```

Comment:
Please prepend this patch with 03-. This generates an implicit order (OmniOS has explicit patch orders by a steering file).

Reply:
I did it and pushed it as a new commit. Thank you.